spec/hydra/pcdm/models/object_spec.rb

```diff
@@ -14,6 +14,45 @@
     end
   end
 
+  context 'when aggregated by other objects' do
+
+    before(:all) do
+      @collection1 = Hydra::PCDM::Collection.create
```

Comment:
Can you shuffle this around a bit so you're using `let` syntax (see lines 5-7 in this file) for creating the collections and objects, and use the `before` block for adding memberships?